app/frontend/components/common/navbar/navbarUnauth.js

```diff
@@ -36,7 +38,7 @@ const NavbarUnauth = connect(
         h(
           'a',
           {
-            class: 'unauth-menu',
+            class: 'navbar-link',
             href: 'https://github.com/vacuumlabs/adalite/wiki/AdaLite-FAQ',
             target: '_blank',
```

Comment:
Add `rel: 'noopener'` to any link that has`target: '_blank'`.

Reply:
will do